src/components/gui/gui.jsx

```diff
@@ -21,6 +21,9 @@ import MenuBar from '../menu-bar/menu-bar.jsx';
 import CostumeLibrary from '../../containers/costume-library.jsx';
 import BackdropLibrary from '../../containers/backdrop-library.jsx';
 import Watermark from '../../containers/watermark.jsx';
+import OldTimeyMode from '../oldtimey-mode/oldtimey-mode.jsx';
```

Comment:
You don't need these, right? Though I guess it doesn't hurt?

Reply:
We may completely remove the code for the other modes in the future, but I think removing the references to them from gui.jsx is a good idea. Done.